src/main/java/no/unit/nva/cristin/projects/Constants.java

```diff
@@ -19,12 +19,10 @@
     public static final String EMPTY_FRAGMENT = null;
     public static final String INSTITUTION_PATH = "institutions";
     public static final String PERSON_PATH = "persons";
-    private static final String BASE_URL_ENV = "BASE_URL";
-    public static final String BASE_URL = ENVIRONMENT.readEnv(BASE_URL_ENV);
+    private static final String BASE_PATH_ENV = "BASE_PATH";
+    public static final String BASE_PATH = ENVIRONMENT.readEnv(BASE_PATH_ENV);
```

Comment:
I suggest that we assign a default value if the value is missing, because in case we do not setup the correct env variables. The reason is that  I am not sure whether things would show up in the logs when errors occur in the static initialization of the code. 
Also, I do not think it is necessary to declare a static variable in this case. Just use the string literal directly.

Reply:
Are we sure default valyes are a good thing, isn't it better to fail fast if something is misconfigured?